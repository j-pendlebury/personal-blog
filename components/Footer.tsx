const Footer = () => {
  return (
    <>
      <div className="gradient-border-line"></div>
      <footer data-testid="footer" id="footer">
        Â© 2021 - Jamie Blaut
      </footer>
    </>
  );
};

export default Footer;
