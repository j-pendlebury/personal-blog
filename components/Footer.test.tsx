import { render } from "@testing-library/react";
import Footer from "./footer";

test("example", () => {
  const { container } = render(<Footer />);

  expect(container.getElementById("footer")).toBeDefined();
  expect(container).toMatchSnapshot();
});
