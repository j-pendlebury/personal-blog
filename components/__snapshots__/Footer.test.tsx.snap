// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example 1`] = `
<div>
  <div
    class="gradient-border-line"
  />
  <footer
    data-testid="footer"
  >
    Â© 2021 - Jamie Blaut
  </footer>
</div>
`;
